### PR TITLE
Refined documentation for loaders

### PR DESCRIPTION
**Pull request by cg-cnu**
Hey guys,

Thanks for the opportunity to work on the docs. 🙂 
As discussed in the pr #12245 I have updated the docs. 
Here is the summary of changes I made in this pr.

* Added a small intro, description of the file format and a link to the wiki page if available.
* Extended the current example and added examples for the ones not present (eg;svg- please review and let me know if its correct).
* Fine tuned explanation for the loader parameters.

I tried my best to keep it clean and consistent. 
If you find any issues let me know, I will fix it.  😄

### CODE REVIEW

File: docs/examples/loaders/SVGLoader.html [hunk 1]
```diff
@@ -11,9 +11,34 @@
...
 
+		// instantiate the loader
+		var loader = new THREE.SVGLoader();
+		
+		// load a SVG resource
+		loader.load(
+			//	resource URL
+			'data/svgSample.svg',
+			// called when the resource is loaded.
+			function(doc){
```
Comment (WestLangley) [inline docs/examples/loaders/SVGLoader.html]: In this code block, and the other code blocks in this PR, you can use http://zz85.github.io/mrdoobapproves/ to help you get the formatting correct.

Comment (cg-cnu) [reply docs/examples/loaders/SVGLoader.html]: Thanks for the input. 👍 
Fixed it. 🙂

Comment (cg-cnu) [reply docs/examples/loaders/SVGLoader.html]: Btw, that's an interesting tool. Much helpful in getting the project requirements out there with out too much documentation and feedback from the project maintainers 😎

Comment (WestLangley) [reply docs/examples/loaders/SVGLoader.html]: Many thanks for your contribution!

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Many thanks!

**Comment by cg-cnu** [comment]
Thank you @mrdoob 😄

**Pull request merged**